code/modules/mob/living/say.dm

```diff
@@ -189,7 +189,15 @@ var/list/headset_modes = list(
 		return
 
 	//parse the language code and consume it
-
+	
+	//but first, scoreboard for syndiphrases stuff
+	if(src.mind.GetRole(TRAITOR) || src.mind.GetRole(NUKE_OP) || src.mind.GetRole(CHALLENGER))
```

Comment:
Mobs that weren't controlled by a player won't have a mind. Won't this runtime when `mind` is null?

Reply:
yes